tests/Integration/Routing/ImplicitBindingTest.php

```diff
@@ -4,13 +4,12 @@
 
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Support\Facades\Route;
 use Illuminate\Support\Facades\Schema;
 use Orchestra\Testbench\TestCase;
 
 class ImplicitBindingTest extends TestCase
 {
-    protected function getEnvironmentSetUp($app)
+    protected function defineEnvironment($app)
     {
```

Comment:
`defineEnvironment()` is the new `getEnvironmentSetUp()` but the old method will be around (not deprecated).